i3_agenda/event.py

```diff
@@ -65,3 +68,46 @@ def default(self, o):  # pylint: disable=E0202
             return o.__dict__
         else:
             return json.JSONEncoder.default(self, o)
+
+
+def get_closest(events: List[Event], hide_event_after: int) -> Optional[Event]:
+    closest = None
+    for event in events:
+        # Don't show all day events
+        if event.is_allday:
+            continue
+
+        now = time.time()
+        # If the event already ended
+        if event.end_time < now:
+            continue
+
+        # If the event started more than hide_event_after ago
+        if hide_event_after > -1 and event.unix_time + 60 * hide_event_after < now:
+            continue
+
+        if closest is None or event.unix_time < closest.unix_time:
+            closest = event
+
+    return closest
+
+
+def get_event_time(full_time: str) -> float:
```

Comment:
I would still prefer it if `get_event_time` and `is_allday` could be Event methods, but I understand that it complicates some stuff quite a bit.  It's okay to leave it this way but I should take a note to get around to it some day

Reply:
Yes, I think now is_allday is solved.

get_event_time I suggest to change this name to get_unix_time, as it is a method that takes the string of the datetime and returns it in unix_time.